projects/pt1/python/torch_mlir_e2e_test/test_suite/argmin.py

```diff
@@ -0,0 +1,72 @@
+#  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
You can move all of these tests to `reduction.py`. [torch.max](https://pytorch.org/docs/stable/generated/torch.max.html) actually returns the index as well (I assume the same is true for min) so testing of related ops is hidden in the reduction test suite.

Reply:
Will do that, then I should move the argmax tests as well which are so far in a separate file.